src/main/scala/libra/ops/base.scala

```diff
@@ -23,7 +26,11 @@ object base {
     * @tparam F the unit to convert from
     * @tparam T the unit to convert to
     */
-  case class ConversionFactor[A, D, F <: UnitOfMeasure[D], T <: UnitOfMeasure[D]](val value: A)
+  case class ConversionFactor[A, D, F <: UnitOfMeasure[D], T <: UnitOfMeasure[D]](val value: A) {
+    def compose[OtherF <: UnitOfMeasure[D]](otherConversionFactor: ConversionFactor[A, D, OtherF, F])
```

Comment:
This is a really nice helper function :+1: 

There was a previous attempt at trying to compose conversion factors implicitly, i.e given an implicit `ConversionFactor[P, Q, A, B]` and an implicit `ConversionFactor[P, Q, B, C]`, produce a `ConversionFactor[P, Q, A, C]`.  However, this couldn't be picked up by scala's implicit resolution.  I'll keep investigating, but this is a good stand in in the meantime.

Reply:
I also tried to make it implicitly but i failed (it was above my abilities), i'll be waiting impatiently for results of your investigation :)

Reply:
I've created an issue to track it properly https://github.com/to-ithaca/libra/issues/50